contracts/Vault.vy

```diff
@@ -1438,12 +1443,14 @@ def removeStrategyFromQueue(strategy: address):
 @internal
 def _debtOutstanding(strategy: address) -> uint256:
     # See note on `debtOutstanding()`.
+    if self.debtRatio == 0:
+        return 0
```

Comment:
This would be better, in case there's a scenario where we set it all to 0 but there's still outstanding debt
```suggestion
        return strategy_totalDebt
```